dpctl-capi/include/dpctl_sycl_context_interface.h

```diff
@@ -161,4 +161,15 @@ DPCTLContext_GetBackend(__dpctl_keep const DPCTLSyclContextRef CtxRef);
 DPCTL_API
 void DPCTLContext_Delete(__dpctl_take DPCTLSyclContextRef CtxRef);
 
+/*!
+ * @brief Wrapper over
+ * std::hash<sycl::context>'s operator()
+ *
+ * @param    CtxRef        The DPCTLSyclContextRef pointer.
+ * @return   Hash value of the underlying sycl::context instance.
```

Comment:
Please wrap the `sycl::context` in double back-ticks? I have started adding double back-tics in docstrings for SYCL API names as it renders better in Sphinx.